private/special_artifacts.bzl

```diff
@@ -0,0 +1,3 @@
+POM_ONLY_ARTIFACTS = {
+    "com.github.fommil.netlib:all": True,
```

Comment:
Does this mean that we need to hardcode all pom only artifacts?

Reply:
Yes. My estimate is that this is a rare occurrence, so we probably shouldn't bake huge custom logic into the rule.